pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -19,7 +19,10 @@
 #define LOAD_CONSTANT(param, var) asm("%0 = " param " ll" \
                                       : "=r"(var))
 
-enum telemetry_counter{tcp_sent_miscounts, missed_tcp_close, udp_send_processed, udp_send_missed};
+enum telemetry_counter { tcp_sent_miscounts,
+    missed_tcp_close,
+    udp_send_processed,
+    udp_send_missed };
```

Comment:
I think we we want `tcp_sent_miscounts` on its own line so that its formatted similarly to the enum in `tcp_states.h`

Reply:
I'll see what option I need to set for it to format that the way we want. 😄